SLING-8170 - Allow Sling servlets to declare a resource super type

raducotescu opened the pull request.

raducotescu committed:
SLING-8170 - Allow Sling servlets to declare a resource super type

kwin commented on src/main/java/org/apache/sling/servlets/annotations/SlingServletResourceTypes.java:
```diff
@@ -43,18 +43,26 @@
+    /**
+     * The resource super type from which the servlet inherits (value is "sling.servlet.resourceSuperType").
+     *
+     * @return the resource super type
```
Can you add a hint from which version of the servlet resolver bundle this is supported and what happens if that is not set (i.e. default handling)

raducotescu replied:
Yes, I was just doing this for the "Servlets and Scripts" site page and was thinking that it would be worth adding this info here too. :)

kwin reviewed (dismissed):
see inline comment

raducotescu committed:
SLING-8170 - Allow Sling servlets to declare a resource super type

* added information about what version of the org.apache.sling.servlets.resolver
bundle needs to be deployed in order for the "sling.servlet.resourceSuperType" to
be taken into consideration
